### ISSUE TITLE
Crash after going to Manage Leftover Data on firsts runs

### ISSUE THREAD

**Issue by pabloab** [flattool/warehouse#46]
Tried Warehouse for the first time on an Ubuntu 20.04. On the first attempt the app crashed after trying to empty all on Manage Leftover Data. On the second attempt, I run on CLI (see below) and the app crash immediately after trying to go to that section. The third attempt everything go as expected.
```sh
$ flatpak run io.github.flattool.Warehouse  # Second run, crash immediately after trying to go to Manage Leftover Data
**
Gtk:ERROR:../gtk/gtkboxlayout.c:646:gtk_box_layout_compute_opposite_size_for_size: assertion failed: (sizes[nvis_children - n_inconstant].data == child)
Bail out! Gtk:ERROR:../gtk/gtkboxlayout.c:646:gtk_box_layout_compute_opposite_size_for_size: assertion failed: (sizes[nvis_children - n_inconstant].data == child)

$ flatpak run io.github.flattool.Warehouse  # Third run

(warehouse:2): Gdk-CRITICAL **: 08:34:22.884: gdk_monitor_get_geometry: assertion 'GDK_IS_MONITOR (monitor)' failed

(warehouse:2): Gdk-CRITICAL **: 08:34:22.896: gdk_monitor_get_geometry: assertion 'GDK_IS_MONITOR (monitor)' failed

(warehouse:2): Gdk-CRITICAL **: 08:34:22.908: gdk_monitor_get_geometry: assertion 'GDK_IS_MONITOR (monitor)' failed

(warehouse:2): Gdk-CRITICAL **: 08:34:23.372: gdk_monitor_get_geometry: assertion 'GDK_IS_MONITOR (monitor)' failed
/usr/lib/python3.11/site-packages/gi/overrides/Gio.py:42: Warning: Source ID 1610 was not found when attempting to remove it
  return Gio.Application.run(self, *args, **kwargs)

$ flatpak run io.github.flattool.Warehouse # On the other runs everything worked fine

$ flatpak run io.github.flattool.Warehous
```

**Comment by heliguy4599** [comment]
Very strange. Looking at these errors it isn't apparent to me what the issue could be, and the fact that it's working perfectly fine on and after the third run makes me think it's just some super bad luck. Not really sure what else to offer here sorry. I'll leave this issue open for a bit if you manager to find anything else out, because I can't reproduce this bug.

**Comment by pabloab** [comment]
I tried on other host, Pop!_OS 22.04, installed for the first time. Everything went fine, even on the first `run`; nothing to stdout. Maybe something particularly with Ubuntu 20.04?

**Comment by heliguy4599** [comment]
While working on a different task I think I might have found what the issue was. In short, I was modifying the subtitle of those rows when there was a rare chance the row did not exists. I have fixed this in a recent commit and that fix will be out in version 1.2.0 (which is releasing very very soon)
